addOns/scanpolicies/src/main/javahelp/help/map.jhm

```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/map_1_0.dtd">
+
+<map version="1.0">
+    <mapID target="scanpolicies" url="contents/scanpolicies.html" />
```

Comment:
I'd add `addon.` prefix to the target keys, makes it less likely to clash with core keys.